src/project-1.js

```diff
@@ -1,139 +1,141 @@
 // Do not change any of the funcxtion names
+// partner: Jonathan Chiang
 
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
-  // code here
+  return num * 10;
 };
 
 const subtractFive = (num) => {
   // return num after subtracting five
-  // code here
+  return num - 5;
 };
 
 const areSameLength = (str1, str2) => {
   // return true if the two strings have the same length
   // otherwise return false
-  // code here
+  return (str1.length === str2.length || false);
 };
 
 const areEqual = (x, y) => {
   // return true if x and y are the same
   // otherwise return false
-  // code here
+  return (x === y || false);
 };
 
 const lessThanNinety = (num) => {
   // return true if num is less than ninety
   // otherwise return false
-  // code here
+  return (num < 90 || false);
 };
 
 const greaterThanFifty = (num) => {
   // return true if num is greater than fifty
   // otherwise return false
-  // code here
+  return (num > 50 || false);
 };
 
 const add = (x, y) => {
   // add x and y together and return the value
-  // code here
+  return x + y;
 };
 
 const subtract = (x, y) => {
   // subtract y from x and return the value
-  // code here
+  return x - y;
 };
 
 const divide = (x, y) => {
   // divide x by y and return the value
-  // code here
+  return x / y;
 };
 
 const multiply = (x, y) => {
   // multiply x by y and return the value
-  // code here
+  return x * y;
 };
 
 const getRemainder = (x, y) => {
   // return the remainder from dividing x by y
-  // code here
+  return x % y;
 };
 
 const isEven = (num) => {
   // return true if num is even
   // otherwise return false
-  // code here
+  return !(num % 2);
```

Comment:
Interesting approach. I'd like to see your thoughts behind coming up with this solution? Was it trial and error? Just found out that num % 2 gives us zero which is an undefined value so you can turn it to it's opposite which is True? Interesting.

Reply:
Some experience in c++ makes me use the NOT operator naturally, and it worked the first time I put it in. Interesting I didn't really think about it being undefined, I thought of it as 0, which in my mind is false